continuationsPlugin/src/test/scala/continuations/StateMachineFixtures.scala

```diff
@@ -1149,22 +1149,16 @@ trait StateMachineFixtures {
        |          var $continuation: continuations.Continuation[Any] | Null = null
        |          completion match
        |            {
-       |              case x$0 @ <empty> if
-       |                x$0.isInstanceOf[
-       |                  continuations.compileFromString$package.
-       |                    compileFromString$package$foo$1
-       |                ].&&(
+       |              case x$0:
+       |                continuations.compileFromString$package.
+       |                  compileFromString$package$foo$1
+       |                if 
        |                  x$0.asInstanceOf[
        |                    continuations.compileFromString$package.
        |                      compileFromString$package$foo$1
        |                  ].$label.&(scala.Int.MinValue).!=(0)
-       |                )
        |               =>
-       |                $continuation =
-       |                  x$0.asInstanceOf[
-       |                    continuations.compileFromString$package.
-       |                      compileFromString$package$foo$1
-       |                  ]
+       |                $continuation =|                  x$0
```

Comment:
This pipe looks like a typo:

```suggestion
       |                $continuation = x$0
```